crates/holochain_state/src/lib.rs

```diff
@@ -1,12 +1,13 @@
+//! @todo: update diagram and types
 //! # Persisted State building blocks
 //!
-//! This crate provides a few types for working with databases. The types build upon those found in [holochain_sqlite::buffer].
-//!
-//! - [ElementBuf]: the union of two CasBuffers, one for Entries, one for Headers
-//! - [ChainSequenceBuf]: database representing the chain sequence DB, which provides a special method for accessing the chain head
-//! - [SourceChainBuf]: the union of a [ElementBuf] and a [ChainSequenceBuf], which fully represents a source chain
-//! - [MetadataBuf]: (*unimplemented*) Uses a KvvBuffer to represent EAV-like relationships between CAS entries
-//! - [Cascade]: (*unimplemented*) Unifies two [ElementBuf] and two [MetadataBuf] references (one of each is a cache) in order to perform the complex metadata-aware queries for getting entries and links, including CRUD resolution
+//! This crate provides a few types for working with databases. The types build upon those found in holochain_sqlite::buffer.
```

Comment:
These docs are really out of date. I don't think the buf types even exist anymore?

Reply:
@freesig That's right, Michael said that too. I've added a `todo` to the docs for a future update.

Reply:
These are so out of date that having no docs would be better than any here. It's a totally different paradigm. We won't even have such a diagram in the future.